bpf/process/types/basic.h

```diff
@@ -2398,6 +2392,12 @@ read_call_arg(void *ctx, struct msg_generic_kprobe *e, int index, int type,
 		arg = (unsigned long)file;
 	}
 		// fallthrough to file_ty
```

Comment:
but now this will rewrite the arg from kiocb_type that is expecting to also  fall through? I think my abuse of fallthroughs has reached its limit of usability? Its a bit tricky to get this good on older kernels. We don't want to inline the code in duplicate chunks repeating here for insns count reasons.

Maybe an inline would get optimized to a goto by compiler so it would be ok? Other option would be a goto to file handler. Anyways as is it needs a fix right?

Reply:
Hmm, if there are no cases where `struct linux_binrpm` wraps a `struct kiocb`, we should be okay, no?

Reply:
Bump @jrfastab. 

`needs-rebase` got added, fully 0 point in doing another rebase (it'll be the 4th since this PR was opened, I think) until I get feedback on this comment.